docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc

```diff
@@ -16,10 +16,10 @@ If you want to enable device config backup on existing nodes in a requisition, s
 . Click on the edit icon and click *Add Node*.
 . Set up your requisition as described in xref:operation:provisioning/getting-started.adoc#requisition-create[Create a requisition].
 . Click *Add Meta-Data*, and define the following key-value pairs:
-.. `username`: username for the device
-.. `password`: password for the device
-.. `script-file`: name of the device configuration script for this type of device
-.. `schedule`: a cron expression to specify the time to fetch the device backup configuration (default is `0 0 0 * * ?`, every 24 hrs at 12:00 a.m.)
+.. `dcb:username`: username for the device
+.. `dcb:password`: password for the device
+.. `dcb:script-file`: name of the device configuration script for this type of device
+.. `dcb:schedule`: a cron expression to specify the time to fetch the device backup configuration (default is `0 0 0 * * ?`, every 24 hrs at 12:00 a.m.)
```

Comment:
Just to verify that this is actually dcb:username? Chandra had said to remove "dcb".

Reply:
@Bonrob2  yes, this is fine. I wasn't sure if we keep the same naming when we implement SCV. 
Also this naming is not something that is hard-coded.  This naming is referred in poller-configuration.xml, So user can update these.

Reply:
Thanks for confirming!